lib/fetch/body.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const Busboy = require('busboy')
```

Comment:
shouldn't busboy be added to package.json as a runtime dependency? it is likely to break in systems without it now.

Reply:
Allowing this to be an optional feature would be nice, although npm has a few issues with optional dependencies. :(

Reply:
@cameron-robey can you add busboy to dependencies?

Reply:
Added in https://github.com/nodejs/undici/pull/1606/commits/0a9785bd7ee28446b551b82b8ab4ba1012bea3c7